test/screenshot/spec/mdc-top-app-bar/classes/baseline-dense-prominent.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
All of these screenshot tests can use mobile viewport, this'll save us some time running these tests. (`.test-viewport--mobile`)

Reply:
Is this what you expect to see? 
![DFUQD9RZXTq](https://user-images.githubusercontent.com/579873/57470327-64ba1900-723d-11e9-981e-5818cf8d3dfb.png)

Reply:
Ideally it should be this size:

Example of tab-bar:

Snapshot:
https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/02/23_48_38_484/spec/mdc-tab-bar/classes/baseline.html?utm_source=golden_json

Screenshot:
https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/02/23_48_38_484/spec/mdc-tab-bar/classes/baseline.html.windows_chrome_73.png

Reply:
Top App Bar in some instances is position:fixed. Do you want me to override this?

Reply:
I think, we can override the width of top app bar when it is fixed.

`test/screenshot/fixture.scss` doesn't seem to export mobile viewport width (350px). Maybe we can export it in a sass variable and reuse it in tab app bar's fixture.

Reply:
I need to change the top app bar to absolute if we want it to appear in the mobile viewport. Is that ok? I don't think we shouldn't be altering the styles in a screenshot test.

Reply:
spoke offline - since short and short collapsed rely on the width property we can only do desktop viewport.

Reply:
Can we not use mobile viewport in remaining tests except for short & short collapsed?